src/admin/content/content.service.ts

```diff
@@ -525,43 +524,44 @@ export class ContentService {
 		profileId: string
 	): Promise<Partial<ContentPageInfo> | null> {
 		try {
-			const contentToInsert = { ...contentPageInfo };
+			const duplicatedContentPage = { ...contentPageInfo };
```

Comment:
Aangezien de functie (`duplicateContentPage`) reeds de context van de entiteit (`ContentPage`) mee geeft ben ik eerder voorstander om dit soort variabelen kort te houden op bv. [the] `const duplicate = ...`.